focus-ios/Blockzilla/Settings/Controller/SettingsViewController.swift

```diff
@@ -480,7 +480,11 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
     }
 
     func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        return sections[section] == .privacy ? 50 : 30
+        return UITableView.automaticDimension
```

Comment:
Since the height is not constant and auto calculated, how does it look on iPad or other devices?

Reply:
Good question. I'm adding some iPad screenshots.

<img src="https://github.com/user-attachments/assets/9cb6a1e5-5718-4144-a6a6-25e4f6b94a56" width=300>
<img src="https://github.com/user-attachments/assets/d9a98cce-5663-4a30-adff-cd804d32bc5c" width=300>